Core/include/Acts/Propagator/DirectNavigator.hpp

```diff
@@ -12,6 +12,7 @@
 #include "Acts/Geometry/Layer.hpp"
 #include "Acts/Geometry/TrackingGeometry.hpp"
 #include "Acts/Geometry/TrackingVolume.hpp"
+#include "Acts/Navigation/DetectorNavigator.hpp"
```

Comment:
```suggestion
```

Reply:
I had to add it due to SYCL complaining on `volInfo`

Reply:
oh I see the function is not defined here but in the other classes / headers. I think we can copy paste it into the private section

from `Navigator.hpp`

```
  template <typename propagator_state_t>
  std::string volInfo(const propagator_state_t& state) const {
    return (state.navigation.currentVolume
                ? state.navigation.currentVolume->volumeName()
                : "No Volume") +
           " | ";
  }
```

Reply:
done. Maybe in future MRs we can make this a global function instead